src/OrbitQt/ConnectToStadiaWidget.cpp

```diff
@@ -435,4 +434,9 @@ void ConnectToStadiaWidget::TrySelectRememberedInstance() {
   }
 }
 
+void ConnectToStadiaWidget::showEvent(QShowEvent* event) {
+  QWidget::showEvent(event);
+  DetachRadioButton();
```

Comment:
This line would benefit from a short explanatory comment. Otherwise this might get removed in a year or so, because nobody remembers.

Reply:
done